CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 
 * chore: Update fastecdsa python package [#1993](https://github.com/lambdaclass/cairo-vm/pull/1993)
 
+* fix: lift `wasm-bindgen =2.92.0` dependency requirement. 
```

Comment:
The previous version of `wasm-bindgen` is `0.2.92` not `2.92.0`. Even so, I would note what's the version this dependency is being updated. Also, add the link to this PR. For example:

```suggestion
* fix: Update wasm-bindgen to version 0.2.100 and unpin its version requirement [#1988](https://github.com/lambdaclass/cairo-vm/pull/1988)
```